<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>loginDialog</class>
 <widget class="QDialog" name="loginDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1234567</width>
    <height>1234567</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="lb_pic">
       <property name="minimumSize">
        <size>
         <width>151</width>
         <height>291</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>1234567</width>
         <height>1234567</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="res.qrc">:/images/register.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="minimumSize">
        <size>
         <width>361</width>
         <height>301</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>1234567</width>
         <height>1234567</height>
        </size>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="page_1">
        <attribute name="title">
         <string>登录</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="lb_phone_log">
           <property name="text">
            <string>手机号</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="le_phone_log">
           <property name="text">
            <string>1</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="lb_password_log">
           <property name="text">
            <string>密码</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="le_password_log">
           <property name="text">
            <string>1</string>
           </property>
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QPushButton" name="pb_log">
           <property name="text">
            <string>登录</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="page2">
        <attribute name="title">
         <string>注册</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="lb_name_reg">
           <property name="text">
            <string>昵称</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="le_name_reg"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="lb_phone_reg">
           <property name="text">
            <string>手机号</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="le_phone_reg"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="lb_password_reg">
           <property name="text">
            <string>密码</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="le_password_reg">
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QPushButton" name="pbreg">
           <property name="text">
            <string>注册</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
